filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -199,10 +202,10 @@ pcl::CropHull<PointT>::applyFilter3D (PointCloud &output)
         crossings[ray] += rayTriangleIntersect
           (input_->points[(*indices_)[index]], rays[ray], hull_polygons_[poly], *hull_cloud_);
 
-    if (crop_outside_ && (crossings[0]&1) + (crossings[1]&1) + (crossings[2]&1) > 1)
-      output.push_back (input_->points[(*indices_)[index]]);
-    else if (!crop_outside_)
+    bool isPointInsideHull = (crossings[0]&1) + (crossings[1]&1) + (crossings[2]&1) > 1;
+    if ((crop_outside_ && isPointInsideHull) || (!crop_outside_ && !isPointInsideHull)) {
```

Comment:
Could this be simplified to `if (crop_outside_ == isPointInsideHull)`?

Reply:
applied the suggestion